### PR TITLE
tile-3d: update with new loaders version

### PR DESCRIPTION
**Pull request by belom88**
#### Background
We updated `Tileset3D` options structure. This update is to meet these changes.
#### Change List
- Fix `Tileset3D` options structure;
- Fix coordinates in test. `throttleRequests` is true by default now and `RequestScheduler` cuts tiles outside of the frustum.

This PR is prior to bumping loaders.gl version to latest beta.

### CODE REVIEW

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js [hunk 1]
```diff
@@ -109,20 +109,25 @@ export default class Tile3DLayer extends CompositeLayer {
...
     if (Array.isArray(loader)) {
       loader = loader[0];
     }
 
-    const options = {...loadOptions};
+    const options = {loadOptions};
     if (loader.preload) {
       const preloadOptions = await loader.preload(tilesetUrl, loadOptions);
+
+      options.loadOptions.fetch = options.loadOptions.fetch || {};
```
Comment (Pessimistress) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:124]: This line is unnecessary unless the next if condition is met

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js [hunk 2]
```diff
@@ -109,20 +109,25 @@ export default class Tile3DLayer extends CompositeLayer {
...
+      if (preloadOptions.headers) {
+        options.loadOptions.fetch.headers = preloadOptions.headers;
```
Comment (Pessimistress) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:126]:
```js
options.loadOptions.fetch = {
  ...options.loadOptions.fetch,
  headers: preloadOptions.headers
}
```

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js [hunk 3]
```diff
@@ -109,20 +109,25 @@ export default class Tile3DLayer extends CompositeLayer {
...
+    const {loadOptions = {}} = this.props;
 
     // TODO: deprecate `loader` in v9.0
     let loader = this.props.loader || this.props.loaders;
     if (Array.isArray(loader)) {
       loader = loader[0];
     }
 
-    const options = {...loadOptions};
+    const options = {loadOptions};
```
Comment (Pessimistress) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:120]: Recommend avoid mutating the user provided `loadOptions` object

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js (hunk 1 above)
Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js]: Done

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js (hunk 2 above)
Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js]: Done

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js (hunk 3 above)
Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js]: Done

File: yarn.lock [hunk 4]
```diff
@@ -1982,6 +1982,20 @@
...
+    "@loaders.gl/core" "3.0.0-beta.5"
+    "@loaders.gl/draco" "3.0.0-beta.5"
+    "@loaders.gl/gltf" "3.0.0-beta.5"
+    "@loaders.gl/loader-utils" "3.0.0-beta.5"
+    "@loaders.gl/math" "3.0.0-beta.5"
+    "@loaders.gl/tiles" "3.0.0-beta.5"
+    "@math.gl/core" "3.5.0-alpha.1"
+    "@math.gl/geospatial" "3.5.0-alpha.1"
+
 "@loaders.gl/core@3.0.0-alpha.21", "@loaders.gl/core@^3.0.0-alpha.21":
```
Comment (Pessimistress) [inline yarn.lock:1985]: Multiple loaders version is installed. Run `yarn list "@loaders.gl/*"` to make sure you have only one copy of each.

Comment (belom88) [reply yarn.lock:1985]: Yeah. I now changed the version only in `geo` module. I wanted to do bumping in another PR.

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js [hunk 5]
```diff
@@ -109,20 +109,27 @@ export default class Tile3DLayer extends CompositeLayer {
...
 
-    const options = {...loadOptions};
+    const options = {loadOptions: {...loadOptions}};
     if (loader.preload) {
       const preloadOptions = await loader.preload(tilesetUrl, loadOptions);
+
+      if (preloadOptions.headers) {
+        options.loadOptions.fetch = {
+          ...options.loadOptions.fetch,
+          headers: preloadOptions.headers
```
Comment (Pessimistress) [inline modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: If `preloadOptions` is supposed to be a top-level loaders.gl options object, then `preloadOptions.headers` should be moved to `preloadOptions.fetch.headers` on the loaders side?

**Review by Pessimistress — APPROVED** [review]

File: modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js (hunk 5 above)
Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: Yeah. It's already done and Cesium tilesets don't work on loaders.gl now.

Comment (Pessimistress) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: I'm saying the `preload` function should return an object that contains `fetch`, in compliance with the latest options format.

Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: Well. I see here
`preloadOptions.headers` -> `options.loadOptions.fetch.headers`.
Anyway it works. I checked with `yarn start-local-deck`

Comment (Pessimistress) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: I know it works, but this change should not be handled by deck.gl. It's not deck's responsibility to fix the format of something that is originated from loaders.gl in order to feed it back to loaders.gl.

Comment (belom88) [reply modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js:127]: This is data that Cesium returns. @ibgreen what do you think if we add some logic in `preload` function?

**Pull request merged**